lib/src/drawConfigContext.dart

```diff
@@ -59,6 +84,8 @@ class DrawConfigContext {
     return _shortURL;
   }
 
+  bool doesNot(bool value) => value == false ? true : false;
```

Comment:
Oh I initially did this, because negations in code always made it slightly harder to read code. For now, I don't need this function in general. But i'll use ! operator from now on :)